apps/passport-client/src/state.ts

```diff
@@ -35,6 +35,7 @@ export interface AppState {
         modalType: "pods-collection";
         activePod?: PCD<unknown, unknown>;
         idType?: "ticketId" | "id";
+        modalGoBackBehavior?: "close" | "back";
```

Comment:
What is this for?

Reply:
We open the pod collection modal with active pod,
When you click on the pods list we want to close them modal on the back btn.
And when you open from the collection modal, we want go back.